### ISSUE TITLE
Quoted-Printable not being handled in Header by LetterparserMail

### ISSUE THREAD

**Issue by thoffmann-fms** [mat-sz/letterparser#14]
Given this:
```
          const result: LetterparserMail = extract(block.Data);
          const raw: LetterparserNode = parse(block.Data);
```
result.from shows
```
address: "jmetertester06@xxx.local"
name: "=?utf-8?Q?jmetertester06@xxx=2Elocal?="
raw: "\"=?utf-8?Q?jmetertester06@xxx=2Elocal?=\" <jmetertester06@xxx.local>"
```
The name is not being converted from Quoted-Printable per the Content-Transfer-Encoding
![image](https://github.com/mat-sz/letterparser/assets/28546239/0bcee8c5-e4b8-41a7-a7a5-4cb2cc5d3e79)

**Comment by mat-sz** [comment]
@thoffmann-fms Could you post the minimal input needed to reproduce the error?

**Comment by thoffmann-fms** [comment]
Does this work?
```
Subject: =?utf-8?Q?Cancelled_Reservation_-_Automation_-_C's_Location;_Sensors;_Sensor2_-_CEDITtest?=
Sender: "=?utf-8?Q?sgmh@xxx=2Elocal?=" <sgmh@xxx.local>
From: "=?utf-8?Q?sgmh@xxx=2Elocal?=" <sgmh@xxx.local>
Date: Wed, 3 Jan 2024 14:32:57 -0500
CC: "=?utf-8?Q?test@test=2Ecom?=" <test@test.com>
To: "=?utf-8?Q?qaautomation@xxxystemsdev=2Eonmicrosoft=2Ecom?=" <qaautomation@xxxystemsdev.onmicrosoft.com>
X-Priority: 3
MIME-Version: 1.0
X-Mailer: JMail 4.5 by Dimac
Content-Type: multipart/mixed;
	boundary="--NEXT_BM_ADD76E77DDAC42499ED36625FEC5B6F6"

This is a multipart message in MIME format.
----NEXT_BM_ADD76E77DDAC42499ED36625FEC5B6F6
Content-Type: text/html;
	charset="utf-8"
Content-Transfer-Encoding: Quoted-Printable
```

**Comment by mat-sz** [comment]
@thoffmann-fms Thank you, fixed in 0.1.4.

**Comment by thoffmann-fms** [comment]
Thanks, I'll check it out.  It may be a few days before I can fully confirm.  There's some specific e-mails that generate that syntax and they only happen periodically in our test environment.
